terraform/warehouse/main.tf

```diff
@@ -194,6 +195,13 @@ resource "fastly_service_vcl" "pypi" {
     placement      = "none"
   }
 
+  logging_datadog {
+    name               = "Log Edge Errors"
+    token              = var.datadog_token
+    response_condition = "Edge Error"
+    format             = "{ \"ddsource\": \"fastly\", \"service\": \"%%{req.service_id}V\", \"date\": \"%%{begin:%Y-%m-%dT%H:%M:%S%z}t\", \"url\": \"%%{json.escape(req.url)}V\", \"message\": \"Edge error occurred!\", \"short_message\": \"edge_error\" }"
```

Comment:
@miketheman do you have input on what format would be most usable here?

Reply:
[Datadog recommends](https://docs.datadoghq.com/integrations/fastly/) using the `Datadog-Fastly` format - https://docs.fastly.com/en/guides/log-streaming-datadog#using-the-json-logging-format (reminder it has to be made a single line and escaped).
Any reason to _not_ use their recommendation?

Reply:
wdyt [efe3d73](https://github.com/pypi/infra/pull/179/commits/efe3d73ea0048c9e4db9497f12c589477476d1d3)?

I dont see any reason not to except it's a lot longer :)

Reply:
Seems like a reasonable (verbose, but hey, it's an error!) log format, and we can add any keys as needed once deployed. Yay JSON!

Reply:
god that is horrible to look at :shipit: